client/tileset/tilespec.cpp

```diff
@@ -1488,8 +1488,8 @@ static void tileset_add_layer(struct tileset *t, mapview_layer layer)
         t, t->city_offset, t->city_flag_offset, t->occupied_offset));
   } break;
   case LAYER_CITY2: {
-    t->layers.emplace_back(
-        std::make_unique<freeciv::layer_city_size>(t, t->city_size_offset));
+    t->layers.emplace_back(std::make_unique<freeciv::layer_city_size>(
```

Comment:
std library, need an include in header?

Reply:
It's only needed in the header if used there (which is the case). But `vector` is somehow magically included everywhere already, I'm not sure which header does that. If you want we can run `iwyu` in another PR...

Reply:
ok, i'll do a fresh compile on clang as a test as well